### ISSUE TITLE
Power systems stopped working with Buildcraft Addditions in jenkins build 

### ISSUE THREAD

**Issue by Lavaflow** [SleepyTrousers/EnderIO#2563]
I was running a new world and trying out BuildCraft Additions early game, and was primarily using that mods power storage, but with EIO power conduits, and everything was working fine (though I had to manually set conduit power input or output, but that's not a big deal), with the latest official release (2.2.8). I then wanted to start using the new inventory manager panel and systems from EIO, so I updated to the EnderIO-1.7.10-2.3.0.390_beta.jar from Jenkins, at which points all EIO power systems stopped interacting with BCA power storage completely. I'm not sure why that broke, and it wasn't that big of a deal for me since I just switched over to using capacitor banks for energy storage, but it might be indicative of a bigger problem, so I wanted to bring it to your attention.

In short; EIO power not working properly with a specific mod in the latest Jenkins build when it works in release version.

**Comment by MatthiasMann** [comment]
Which version of Buildcraft Additions?

**Comment by Lavaflow** [comment]
I'm using BuildcraftAdditions-1.7.10-2.1.4.jar. I believe it's the latest (or at least was last weekend). 
Buildcraft version is 6.4.16, in case that matters, and bdlib is 1.8.1.87.
I also have Buildcraft-compat-6.4.2, which probably doesn't matter but might.

I should also mention that I'm using fastcraft 1.22-ctest8, which may make a difference. 

When I have time later today/tonight I'll check for a buildcraft update and test if the release version of fastcraft changes things. I don't expect it will, but checking is good.

**Comment by AEnterprise** [comment]
was there a change in the cofh rf api?

**Comment by tterrag1098** [comment]
Nothing that I know of

**Comment by Lavaflow** [comment]
I should have also mentioned I'm currently using forge 10.13.3.1428
I didn't create a new instance to test, but i did use a fresh creative world. FML says I have 240 loaded on the main menu, so there might be another mod interaction somewhere that's being buggy and somehow causing this.

Results/test log:
COFH core, cofh lib, and thermal foundation all up to date. 
Updating to buildcraft v. 6.4.17 did not fix
Tried updating to bc 7.0.8 and bc-compat 7.0.7 (latest official at current time) but game didn't start because logistics pipes didn't like it. (not that I got a crash report, this is why I always have the launcher game output on)
removed logistics pipes to get the game to start. 
Got a crash report on game startup about buildcraft creative tab.
I'm not going to create a new instance to test BC now. Not happening.
reverted to bc 6.4.17 and bc-compat 6.4.2 and reinstalled logistics pipes
reverted to fastcraft 1.21; still not working
unreverting back up to fastcraft 1.22.ctest8
updating to forge 10.13.4.1448; no effect 


In short, none of the mod that are most likely to affect the interaction somehow have an easy fix

**Comment by AEnterprise** [comment]
i have no idea what could be the cause of this, i have not touched this code recently and it heapened afther updating EiO so i have no idea of the possible cause

**Comment by tterrag1098** [comment]
@AEnterprise This is almost surely on your end, though I can't imagine how. Your best best is probably installing EIO in your dev environment and running the debugger.

**Comment by AEnterprise** [comment]
yeah, got a report of this with another mod then enderIO though i don't have any idea of the cause or ways to reproduce it

**Comment by tterrag1098** [comment]
Put a breakpoint on the canConnectEnergy method (my first guess)

**Comment by AEnterprise** [comment]
problem is that in my dev world it works and they all say that breaking and replacing the structure fixes it

**Comment by Lavaflow** [comment]
I made a new instance with only these (http://imgur.com/fIDf7pA) mods in forge 1448, created a test world, just to see if it was any other mod doing something weird somehow.
I put an EIO Stirling generator next to a kinetic energy buffer, and it didn't transfer power, even after setting power output direction in the generator config.

**Comment by tterrag1098** [comment]
Why CoFHCore? That is not needed.

**Comment by Lavaflow** [comment]
I thought it was a req for something. I removed it and tried again.
no change
also removed cofhlib, also didn't fix it.

**Comment by AEnterprise** [comment]
@Lavaflow did you use a machine configurator to make sure the KEB accepts power on that side?

**Comment by Lavaflow** [comment]
I... didn't realize that was a thing.
But it was on both for all sides, so I switched it to input on all sides. Didn't make a difference.

**Comment by AEnterprise** [comment]
i'll see if i can track this down from my end
